src/pages/profile.tsx

```diff
@@ -7,8 +7,12 @@ import { useContext, useEffect, useState } from "react";
 import EditProfile from "../components/EditProfile";
 import ProfileStats from "../components/ProfileStats";
 import { getUserPostsLimit } from "@/firebase/functions";
-
+import ProfileHeader from "../components/profile/ProfileHeader"
+import noAvatar from 'public/noAvatar.jpeg'
+import AboutSection from '../components/profile/AboutSection'
+import PhotoGrid from "../components/profile/PhotoGrid"
 export default function Profile() {
```

Comment:
this profile page is way worse than the one on main, please just add a button that lets you change the profile picture, nothing else for the profile page really needs to be changed